src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabaseTaskHelper.cs

```diff
@@ -168,8 +165,7 @@ private static void AddAzureProperties(DatabaseInfo databaseInfo, DatabaseProtot
 
         private static string GetBackupDate(DateTime backupDate)
         {
-            if (backupDate == null
-                || backupDate < minBackupDate)
+            if (backupDate < minBackupDate)
```

Comment:
> ##[warning]src\Microsoft.SqlTools.ServiceLayer\Admin\Database\DatabaseTaskHelper.cs(171,17): Warning CS8073: The result of the expression is always 'false' since a value of type 'DateTime' is never equal to 'null' of type 'DateTime?'